src/it/scala/io/scalac/tezos/translator/LibrarySpec.scala

```diff
@@ -244,63 +271,74 @@ class LibrarySpec
+    Put(s"$libraryEndpoint?uid=17976f3a-505b-4d66-854a-243a70bb94c0&status=declined")
+      .withHeaders(Authorization(OAuth2BearerToken(bearerToken))) ~> libraryRoute ~> check {
       status shouldBe StatusCodes.OK
     }
     // record3
-    Put(s"$libraryEndpoint?uid=5d8face2-ab24-49e0-b792-a0b99a031645&status=pending_approval").withHeaders(Authorization(OAuth2BearerToken(bearerToken))) ~> libraryRoute ~> check {
+    Put(s"$libraryEndpoint?uid=5d8face2-ab24-49e0-b792-a0b99a031645&status=pending_approval")
+      .withHeaders(Authorization(OAuth2BearerToken(bearerToken))) ~> libraryRoute ~> check {
       status shouldBe StatusCodes.NotFound // cannot update status to "pending_approval"
     }
 
     // invalid uid
-    Put(s"$libraryEndpoint?uid=aada8ebe&status=accepted").withHeaders(Authorization(OAuth2BearerToken(bearerToken))) ~> libraryRoute ~> check {
+    Put(s"$libraryEndpoint?uid=aada8ebe&status=accepted")
+      .withHeaders(Authorization(OAuth2BearerToken(bearerToken))) ~> libraryRoute ~> check {
       status shouldBe StatusCodes.BadRequest
     }
     // non exisitng uid
-    Put(s"$libraryEndpoint?uid=4cb9f377-718c-4d5d-be0d-118a5c99e298&status=accepted").withHeaders(Authorization(OAuth2BearerToken(bearerToken))) ~> libraryRoute ~> check {
+    Put(s"$libraryEndpoint?uid=4cb9f377-718c-4d5d-be0d-118a5c99e298&status=accepted")
+      .withHeaders(Authorization(OAuth2BearerToken(bearerToken))) ~> libraryRoute ~> check {
       status shouldBe StatusCodes.NotFound
     }
 
-
     val expectedNewStatuses = Seq(record1.copy(status = Accepted), record2.copy(status = Declined), record3)
     whenReady(libraryService.getRecords(limit = Some(Limit(refineMV[Positive](5))))) {
       _ should contain theSameElementsAs expectedNewStatuses
     }
 
-    val expectedRecord1 = LibraryEntryRoutesDto(
-      Title(refineMV[NotEmptyAndNotLong]("nameE1")),
-      Author(refineMV[NotEmptyAndNotLong]("authorE1")).some,
-      None,
-      Description(refineMV[NotEmptyAndNotLong]("descriptionE1")).some,
-      Micheline(refineMV[NonEmpty]("michelineE1")),
-      Michelson(refineMV[NonEmpty]("michelsonE1")))
+    val expectedRecord1 = LibraryEntryRoutesDto(libraryEntryId,
+                                                Title(refineMV[NotEmptyAndNotLong]("nameE1")),
+                                                None,
+                                                None,
+                                                None,
+                                                Micheline(refineMV[NonEmpty]("michelineE1")),
+                                                Michelson(refineMV[NonEmpty]("michelsonE1")))
 
     Get(libraryEndpoint) ~> libraryRoute ~> check {
       status shouldBe StatusCodes.OK
       val actualRecords = responseAs[List[LibraryEntryRoutesDto]]
-      actualRecords should contain theSameElementsAs Seq(expectedRecord1)
+      actualRecords.head.copy(uid = libraryEntryId) shouldBe expectedRecord1.copy(uid = libraryEntryId)
```

Comment:
please revise if copying libraryEntryId is needed

Reply:
copy deleted but  test is still not good